.github/workflows/Publish-Package.yml

```
@@ -0,0 +1,57 @@
+name: Publish Package
+
+on:
+  push:
+    branches: [master]
+    tags:
+      + "v[0-9]+.[0-9]+.[0-9]+*" # Push events to matching v*, i.e. v1.0.0, v20.15.10
```

Do you need to escape the dots?
```suggestion
      + "v[0-9]+\.[0-9]+\.[0-9]+*" # Push events to matching v*, i.e. v1.0.0, v20.15.10
```

---

`tags` uses wildcards to match where `.` is not a special character https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet